Services/EntityMerger.php

```diff
@@ -15,41 +16,89 @@ class EntityMerger
     private $tokenStorage;
 
     private $requestStack;
+    
+    private $manager;
+    
+    private $defaultLocale;
+    
+    private $retrieveAllLanguages;
 
     public function __construct(
         Translator $translator,
         TokenStorage $tokenStorage,
-        RequestStack $requestStack
+        RequestStack $requestStack,
+        EntityManager $manager
     ) {
         $this->translator = $translator;
         $this->tokenStorage = $tokenStorage;
         $this->requestStack = $requestStack;
+        $this->manager = $manager;
+        $this->defaultLocale = null;
+        $this->retrieveAllLanguages = false;
     }
+    
+    public function retrieveAllLanguages() {
+        $this->retrieveAllLanguages = true;
```

Comment:
essendo un booleano userei il pattern (is|are)XXXXX del tipo areAllLanguagesRequested